<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>930</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="LSQueueWidget">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>301</width>
      <height>261</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="rowCount">
     <number>6</number>
    </property>
    <property name="columnCount">
     <number>2</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
   </widget>
   <widget class="QTableWidget" name="stationWidget">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>561</width>
      <height>241</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="rowCount">
     <number>5</number>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row/>
    <row/>
    <row/>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QTableWidget" name="registerWidget">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>701</y>
      <width>901</width>
      <height>141</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="cornerButtonEnabled">
     <bool>true</bool>
    </property>
    <property name="rowCount">
     <number>2</number>
    </property>
    <property name="columnCount">
     <number>11</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <row/>
    <row/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <column/>
    <item row="0" column="0">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="6">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="7">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="8">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="9">
     <property name="text">
      <string/>
     </property>
    </item>
    <item row="0" column="10">
     <property name="text">
      <string/>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1066</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="ToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="addAction"/>
   <addaction name="loadAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="addAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/src/src/add.jpg</normaloff>:/src/src/add.jpg</iconset>
   </property>
   <property name="text">
    <string>添加指令</string>
   </property>
  </action>
  <action name="loadAction">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/src/src/load.png</normaloff>:/src/src/load.png</iconset>
   </property>
   <property name="text">
    <string>从文件载入</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
